### ISSUE TITLE
Video overlay doesn't work from exe (Trac #65)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#69]
The video overlay function works when the program is run through the python interpreter, but not from the pyinstaller generated .exe.

Tested on version v0.1-384-gfcc3177, WinXP and Win7.

Migrated from http://microfluidics.utoronto.ca/ticket/65
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "The video overlay function works when the program is run through the python interpreter, but not from the pyinstaller generated .exe.\n\nTested on version v0.1-384-gfcc3177, WinXP and Win7.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Video overlay doesn't work from exe", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-09T14:27:44", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120309T14:31:53: ryan changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20120312T18:22:22: 
* cfobel commented:

> This is fixed by commit 206741e.
> It was caused by a PIL import error, in turn causing camera capture to be disabled.


* cfobel changed resolution from "" to "fixed"
* cfobel changed status from "assigned" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "gui" to "microdrop"
